requirements.txt

```diff
@@ -1,6 +1,7 @@
-requests==2.20.0
+git+git://github.com/radome/ols-client@EP-4829_docker_test#egg=ebi-ols-client
+requests==2.22.0
 SQLAlchemy==1.2.11
-ebi-ols-client==1.0.8
+# ebi-ols-client==1.0.8
```

Comment:
May be we should merge ols-client to master first, and set this to Ensembl repo ?

Reply:
I agree. Also note that I bumped ols-client version [here](https://github.com/Ensembl/ols-client/pull/21/files#diff-021321e8c168ba3ae39ce3a2e7b3ec87)

Reply:
Can't see the bump, actually was thinking of having a direct dependency against repo instead. What's your opinion about this ?

Reply:
I'd leave it on PyPI. We could automate publishing when creating new releases on GitHub.

Reply:
Agreed then. We'll need a "ensprod" user somehow to automate this, I presume.

Reply:
Yep